helmcharts/garden-kube-apiserver/templates/deployment-kube-apiserver.yaml

```diff
@@ -122,6 +122,13 @@ spec:
         - --service-account-issuer=https://kubernetes.default.svc
         - --service-account-signing-key-file=/srv/kubernetes/service-account-key/service_account.key
         - --service-account-key-file=/srv/kubernetes/service-account-key/service_account.key
+        {{- if .Values.apiServer.featureGates }}
```

Comment:
Can we check if this is not possible now via the upstream chart?

Reply:
Si Senor, lemme check

Reply:
Well I don't see anything which makes feature gates configurable. Not even in latest: https://github.com/YAKEcloud/yake/blob/release-v1.93/helmcharts/garden-kube-apiserver/templates/deployment-kube-apiserver.yaml

Reply:
Or am I blind?

Reply:
correct! maybe we should contribute it